packages/property-grid/src/PropertyGridUiItemsProvider.tsx

```diff
@@ -3,46 +3,47 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
-import { connect } from "react-redux";
-
-import { I18N } from "@bentley/imodeljs-i18n";
 import {
   AbstractWidgetProps,
   StagePanelLocation,
   StagePanelSection,
+  StageUsage,
   UiItemsProvider,
+  WidgetState,
 } from "@bentley/ui-abstract";
-import { FrameworkState } from "@bentley/ui-framework";
-import { PropertyGrid, PropertyGridProps } from "./PropertyGrid";
+import { PropertyGridWidgetBaseProps } from "./types";
 import * as React from "react";
-
-// Map framework state to property grid props
-const mapStateToProps = (state: FrameworkState): PropertyGridProps => {
-  return {
-    iModelConnection: state.sessionState.iModelConnection,
-    projectId: state.sessionState.iModelConnection.iModelToken.contextId,
-  };
-};
-
-// tslint:disable-next-line: variable-name
-const ConnectedPropertyGrid = connect(mapStateToProps)(PropertyGrid);
+import { FunctionalPropertyGridWidget } from "./components/FunctionalPropertyGridWidget";
 
 /** Provides the property grid widget to zone 9 */
 export class PropertyGridUiItemsProvider implements UiItemsProvider {
-  public readonly id = "PropertyGridUiitemsProvider";
-  public static i18n: I18N;
+  public readonly id = "PropertyGridUiItemsProvider";
+
+  private _props?: Partial<PropertyGridWidgetBaseProps>;
+
+  constructor(props?: Partial<PropertyGridWidgetBaseProps>) {
+    this._props = props;
+  }
 
   public provideWidgets(
     _stageId: string,
-    _stageUsage: string,
+    stageUsage: string,
     location: StagePanelLocation,
-    _section: StagePanelSection | undefined,
+    section?: StagePanelSection | undefined
   ): ReadonlyArray<AbstractWidgetProps> {
     const widgets: AbstractWidgetProps[] = [];
-    if (location === StagePanelLocation.Right) {
+    if (
+      stageUsage === StageUsage.General &&
+      location === StagePanelLocation.Right &&
+      section === StagePanelSection.End
+    ) {
       widgets.push({
         id: "propertyGrid",
```

Comment:
The id should be unique across all item providers so maybe use 'vcr:PropertyGrid'